tasks/main.yml

```
@@ -35,6 +35,29 @@
   with_items: "{{ jumpbox_operators }}"
   register: user_account
 
+- name: Determine existing users
+  shell: 'executable=/bin/bash set -o pipefail && grep operators /etc/group | cut -d: -f4 | tr "," "\n"'
```

Minor, but I prefer to write this like:

```suggestion
  shell: |
    set -o pipefail
    grep operators /etc/group | cut -d: -f4 | tr "," "\n"'
  args:
    executable: /bin/bash
```